### PR TITLE
Update to phoenix_live_view version 0.9.0 for #5

### PR DESCRIPTION
**Pull request by nelsonic**
This Pull Request has only one goal: 

+ [x] Update the version of `phoenix_live_view` to `0.9.0` (_released 2 days ago_) #5

### CODE REVIEW

**Comment by nelsonic** [comment]
So far this has proved elusive. 😞 

Using version `0.8.1` the counter works flawlessly: 


But when we attempt to update to the _latest_ version `0.9.0` the layout template fails to render:
image

Which means none of the LiveView functionality works! 💔 
I'm going to bit more research and figure out what needs to be changed. 🔍 
BRB! ⏳

**Comment by nelsonic** [comment]
@SimonLab I've managed to get the counter app working with LiveView `v.0.9.0` 🎉 
The code changes are actually quite minimal, but I had to make a few updates to the `README.md` specifically with links to the code snapshots in light of the changes. 🙄 
Please review/merge when you can. (_not urgent_) 👍 
Thanks! ✨

File: README.md [hunk 1]
```diff
@@ -362,7 +365,7 @@ defp deps do
     {:gettext, "~> 0.11"},
     {:jason, "~> 1.0"},
     {:plug_cowboy, "~> 2.0"},
-    {:phoenix_live_view, "~> 0.8.1"},
+    {:phoenix_live_view, "~> 0.9.0"},
```
Comment (SimonLab) [inline README.md]: The explanation above needs also to be changed:

```
Append the following line to the end of the list:
```
{:phoenix_live_view, "~> 0.9.0"},
```

Comment (SimonLab) [reply README.md]: The references to the code need also to be updated to link the lines of code to this PR:

> 🏁 [`mix.exs`](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/d0521ced8cb9d55a6a214a886ecab811ee065960/mix.exs#L33-L46)
file at the end of Step 2:
[mix.exs#L44](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/d0521ced8cb9d55a6a214a886ecab811ee065960/mix.exs#L44)

Comment (nelsonic) [reply README.md]: @SimonLab good catch! Thanks for updating the link. 👍

Comment (nelsonic) [reply README.md]: This should now be resolved: https://github.com/dwyl/phoenix-liveview-counter-tutorial/pull/6/commits/58a105ced960a078dc911c64157827f9290acc73

**Pull request merged**